lib/Data.js

```diff
@@ -30,22 +30,32 @@ export class Suite {
     }
 
     get runtime() {
-        return this.getAllTests()
+        var runtime = this.getAllTests()
             .map((x) => x.runtime)
             .reduce((sum, x) => sum + x, 0);
+        if (isNaN(runtime)) {
```

Comment:
Under what condition do we end up with NaN? Can we detect that condition earlier?